src/test/java/codeu/controller/ActivityFeedServletTest.java

```diff
@@ -21,17 +21,17 @@
 
   @Before
   public void setup() throws IOException {
-    activityfeedServlet = new ActivityFeedServlet(); 
-    mockRequest = Mockito.mock(HttpServletRequest.class);
-    mockResponse = Mockito.mock(HttpServletResponse.class);
-    mockOutputStream = Mockito.mock(ServletOutputStream.class);
-    Mockito.when(mockResponse.getOutputStream())
-    .thenReturn(mockOutputStream);
+  activityfeedServlet = new ActivityFeedServlet(); 
+  mockRequest = Mockito.mock(HttpServletRequest.class);
+  mockResponse = Mockito.mock(HttpServletResponse.class);
+  mockOutputStream = Mockito.mock(ServletOutputStream.class);
+  Mockito.when(mockResponse.getOutputStream())
+        .thenReturn(mockOutputStream);
   }
-	
-  @Test
+  /*@Test
```

Comment:
Why comment out this test?

Reply:
I commented out the test because it was an old test for when I had the prototype

Reply:
Okay. Should update the test to work for the code as it is written now, otherwise remove it.